### ISSUE TITLE
TypeError on Windows 

### ISSUE THREAD

**Issue by nizienko** [Jtalk/url-health-check-action#7]
Hello. After switching from v1.5 to v2 have got an error on Windows github machine:
```
      - name: Wait for Idea started
        uses: jtalk/url-health-check-action@v2
        with:
          url: http://localhost:8082
          max-attempts: 15
          retry-delay: 30s
```

```
Run jtalk/url-health-check-action@v2
  with:
    url: http://localhost:8082
    max-attempts: 15
    retry-delay: 30s
    follow-redirect: false
    retry-all: false
  env:
    JAVA_HOME: C:\hostedtoolcache\windows\Java_Zulu_jdk\11.0.13-8\x64
Checking http://localhost:8082
(node:4444) UnhandledPromiseRejectionWarning: TypeError: arg.includes is not a function
    at ToolRunner._uvQuoteCmdArg (D:\a\_actions\jtalk\url-health-check-action\v2\dist\index.js:923:18)
    at ToolRunner._windowsQuoteCmdArg (D:\a\_actions\jtalk\url-health-check-action\v2\dist\index.js:774:25)
    at ToolRunner._getCommandString (D:\a\_actions\jtalk\url-health-check-action\v2\dist\index.js:705:37)
    at ToolRunner.<anonymous> (D:\a\_actions\jtalk\url-health-check-action\v2\dist\index.js:1026:57)
    at Generator.next (<anonymous>)
    at D:\a\_actions\jtalk\url-health-check-action\v2\dist\index.js:650:71
    at new Promise (<anonymous>)
    at __webpack_modules__.8159.__awaiter (D:\a\_actions\jtalk\url-health-check-action\v2\dist\index.js:646:12)
    at D:\a\_actions\jtalk\url-health-check-action\v2\dist\index.js:1018:53
    at new Promise (<anonymous>)
(node:4444) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:4444) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by Jtalk** [comment]
Hey @nizienko, thanks for getting in touch.

This is the most fascinating issue. 

The problem itself was fairly trivial: Github Toolkit's `exec` doesn't like numbers when it comes to windows.

But the real question was why didn't the tests pick it up, even though your combination of options seemed to have been covered?

It turns out, Actions do not monitor unhandled promise rejections. An unhandled promise rejection, which happens in this case, simply gets ignored and the step gets marked as passed. 

Fixing this also unearthed the problem with the built-in Curl upgrade on... windows. 

But all's well, both have been fixed, both v2 and v2.1 should be pointing at the version that works.

**Comment by nizienko** [comment]
Great, thanks for fixing this so quickly!
